File: Reverse a String[27-August-2023]/Solution.java
//  >> JAVA CODE <<

class Reverse
{
    // Complete the function
    // str: input string
    public static String reverseWord(String str)
    {
        StringBuffer sb=new StringBuffer("");
        
        for(int i=str.length()-1;i>=0;i--)
        {
            sb.append(str.charAt(i));
        }
        
        return sb.toString();
    }
}

File: Reverse a String[27-August-2023]/Solution.cpp
//  >> C++ CODE <<

class Solution
{
    public:
    string reverseWord(string str)
    {
        string ans="";
        
        for(int i=str.length()-1;i>=0;i--)
        {
            ans+=str[i];
        }
        
            return ans;
    }
};